librz/analysis/p/analysis_6502_cs.c

```diff
@@ -23,37 +23,37 @@ static int analop(RzAnalysis *a, RzAnalysisOp *op, ut64 addr, const ut8 *buf, in
 #if USE_ITERZ_API
 	static
 #endif
-	cs_insn *insn = NULL;
+		cs_insn *insn = NULL;
```

Comment:
Why is it doing this?

Reply:
probably because of the `static` modifier above...

Reply:
Omg, let's please remove this static!

Reply:
USE_ITERZ_API is anyway not defined afais. but i would not do it in this pr. I think it's important we keep this big style change clean, without irrelevant changes, just style.

Reply:
Yeah one more indent here doesn't matter.